mirage/scenarios/collections.ts

```diff
@@ -124,18 +134,116 @@ export function collectionAcceptedScenario(server: Server, currentUser: ModelIns
         creator: currentUser,
-        guid: server.create('node', 'withContributors'),
+        guid: nodeAdded,
+        id: nodeAdded.id,
+        reviewsState: CollectionSubmissionReviewStates.Rejected,
+        collection: primaryCollection,
+    });
+    const nodeAccepted = server.create('node', {
+        description: 'An acceped project on the Rejected Collection Example',
+        title: 'Accepted Project on the Rejected Collection Example',
+        license: licensesAcceptable[0],
+        currentUserPermissions: Object.values(Permission),
+    });
+    server.create('contributor', {
+        node: nodeAccepted,
+        users: currentUser,
+        index: 0,
+    });
+    server.create('collection-submission', {
+        creator: currentUser,
+        guid: nodeAccepted,
+        id: nodeAccepted.id,
         collection: primaryCollection,
     });
+    server.create('collection-provider', {
+        id: 'rejected-collection-example',
+        primaryCollection,
+        licensesAcceptable,
+    });
+}
+
+export function collectionRemovedScenario(server: Server, currentUser: ModelInstance<User>) {
+    const licensesAcceptable = server.schema.licenses.all().models;
+    const primaryCollection = server.create('collection');
+    const nodeToBeAdded = server.create('node', {
+        title: 'Removed Node on a collection',
+        currentUserPermissions: Object.values(Permission),
+    });
+    server.create('contributor', {
+        node: nodeToBeAdded,
+        users: currentUser,
+        index: 0,
+    });
+    const nodeAdded = server.create('node', {
+        description: 'A removed project on the Removed Collection Example',
+        title: 'Removed Project',
+        license: licensesAcceptable[1],
+        currentUserPermissions: Object.values(Permission),
+    });
+    server.create('contributor', {
+        node: nodeAdded,
+        users: currentUser,
+        index: 0,
+    });
     server.create('collection-submission', {
         creator: currentUser,
-        guid: server.create('node', 'withContributors'),
+        guid: nodeAdded,
+        id: nodeAdded.id,
+        reviewsState: CollectionSubmissionReviewStates.Rejected,
+        collection: primaryCollection,
+    });
+    const nodeAccepted = server.create('node', {
+        description: 'An acceped project on the Removed Collection Example',
+        title: 'Accepted Project on the Removed Collection Example',
+        license: licensesAcceptable[0],
+        currentUserPermissions: Object.values(Permission),
+    });
+    server.create('contributor', {
+        node: nodeAccepted,
+        users: currentUser,
+        index: 0,
+    });
+    server.create('collection-submission', {
+        creator: currentUser,
+        guid: nodeAccepted,
+        id: nodeAccepted.id,
         collection: primaryCollection,
     });
     server.create('collection-provider', {
-        id: 'accepted-collection-example',
+        id: 'removed-collection-example',
```

Comment:
Can we use the same collection-provider and the same collection for each of these states? Having to change to a different collection-provider to see submissions in a different state seems clunky to me